web/src/components/core/ValidationErrors.jsx

```diff
@@ -22,45 +22,28 @@
 // @ts-check
 
 import React, { useState } from "react";
-import {
-  Button,
-  List,
-  ListItem,
-  Popover
-} from "@patternfly/react-core";
 import { sprintf } from "sprintf-js";
 
 import { Icon } from '~/components/layout';
-import { _, n_ } from "~/i18n";
+import { IssuesDialog } from "~/components/core";
+import { n_ } from "~/i18n";
 
 /**
- * @param {import("~/client/mixins").ValidationError[]} errors - Validation errors
- * @return React.JSX
- */
-const popoverContent = (errors) => {
-  const items = errors.map((e, i) => <ListItem key={i}>{e.message}</ListItem>);
-  return (
-    <List>{items}</List>
-  );
-};
-
-/**
- * Displays a list of validation errors
+ * Displays validation errors
  *
  * When there is only one error, it displays its message. Otherwise, it displays a generic message
- * and the details in an Popover component.
+ * which can be clicked and more error details will be shown in a popup dialog.
  *
  * @component
  *
  * @todo This component might be more generic.
- * @todo Improve the contents of the Popover (e.g., using a list)
  *
  * @param {object} props
- * @param {string} props.title - A title for the Popover
+ * @param {string} props.sectionId - Name of the section which is displaying errors. (product, software, storage, ...)
  * @param {import("~/client/mixins").ValidationError[]} props.errors - Validation errors
  */
-const ValidationErrors = ({ title = _("Errors"), errors }) => {
-  const [popoverVisible, setPopoverVisible] = useState(false);
+const ValidationErrors = ({ errors, sectionId }) => {
+  const [showIssuesPopUp, setshowIssuesPopUp] = useState(false);
```

Comment:
NP: better to use camelCase all the time, 

```suggestion
  const [showIssuesPopUp, setShowIssuesPopUp] = useState(false);
```

Spotted by the spell checker, https://github.com/openSUSE/agama/actions/runs/7364595390/job/20090333162?pr=886#step:6:12

Which you can run locally with `npm run cspell` if you want.